wallet/src/Pos/Wallet/Web/Tracking/Modifier.hs

```diff
@@ -103,6 +103,15 @@ type CachedCAccModifier = CAccModifier
 -- Funcs
 ----------------------------------------------------------------------------
 
+deleteNotDeep :: (Eq k, Hashable k) => k -> MapModifier k v -> MapModifier k v
```

Comment:
This function would benefit from a top-level comment to explain what "deep" means in the context.